libopendavinci/include/opendavinci/odtools/player/Player.h

```diff
@@ -52,7 +79,7 @@ namespace odtools {
                  *
                  * @param obj Reference to an object of this class.
                  */
-                Player(const Player &/*obj*/);
+                Player(const Player &/*obj*/) = delete;
```

Comment:
These can be put as public, I think it's nice to have them directly after the normal constructor, to directly indicate how and how not this object can be created.